src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BooksList from '../containers/BooksList';
+import BooksForm from '../containers/BooksForm';
+
+function App() {
+  return (
+    <div>
+      <BooksList key={12345} />
```

Comment:
- Why the `key` attribute is being assigned here? Is this necessary :thinking: ?